src/Symfony/Component/ErrorCatcher/DependencyInjection/LazyLoadingErrorRenderer.php

```diff
@@ -12,14 +12,14 @@
 namespace Symfony\Component\ErrorCatcher\DependencyInjection;
 
 use Psr\Container\ContainerInterface;
-use Symfony\Component\ErrorCatcher\ErrorRenderer\ErrorRenderer as BaseErrorRenderer;
+use Symfony\Component\ErrorCatcher\ErrorRenderer\ErrorRenderer;
 
 /**
  * Lazily loads error renderers from the dependency injection container.
  *
  * @author Yonel Ceruto <yonelceruto@gmail.com>
  */
-class ErrorRenderer extends BaseErrorRenderer
+class LazyLoadingErrorRenderer extends ErrorRenderer
```

Comment:
makes the classes more understandable and is consistent with `LazyLoadingFragmentHandler`